src/stan-users-guide/simple_linear_regression/run.R

```diff
@@ -0,0 +1,32 @@
+library("cmdstanr")
+
+#set_cmdstan_path(paste(Sys.getenv("HOME"),"/.cmdstanr/cmdstan",sep=''))
+
+# Run from command line: Rscript regression_1.R
+# If running from RStudio remember to set the working directory 
+# >Session>Set Working Directory>To Source File Location
+
+#generate data
+alpha <- 2
+beta <- 3
+sigma <- 1
+N <- 100
+x <- runif(N, 0, 10)
+y <- rnorm(N, alpha + beta * x, sigma)
+
+stan_data <- list(N = N, x = x, y = y)
+
+print(sprintf("generating parameters are: alpha=%.1f, beta=%.1f, sigma=%.1f, N=%d",
```

Comment:
This is going to be a pain if we try to do this in general when the parameter sizes get big.  Also, it's a big maintenance effort with naming.  I'd just skip the printing of the simulated parameters by default.  A much better output in general (that someone like @jgabry could figure out) would be to line up simulated with posterior means/medians/intervals.

Reply:
I can mirror the estimates with the generating params easily enough. I'll ask Jonah if he has better ideas.